package/src/components/MessageList/MessageList.tsx

```diff
@@ -805,13 +805,15 @@ const MessageListWithContext = <
 
   const handleScroll: ScrollViewProps['onScroll'] = (event) => {
     const offset = event.nativeEvent.contentOffset.y;
+    const messageListHasMessages = channel.state.messages.length > 0;
```

Comment:
how about thread list here? a thread list can also be empty right ?

you have to check this via the thread screen.. when navigating to a thread screen directly `channel.state.messages.length` has no relation.. 

so then we have to check the length of the message list that is used to render I believe

Reply:
Changed 👍🏻